.drone.yml

```diff
@@ -119,6 +119,14 @@ steps:
   - codespell
   - shellcheck
 
+- name: lighthouse
+  image: grafana/build-container:1.2.28
+  commands:
+  - lhci autorun || echo "LHCI failed!"
```

Comment:
There must be a bug here, since `lhci` doesn't exist. Is it supposed to be in ./node_modules/.bin?